config/routes.rb

```diff
@@ -28,6 +28,7 @@
     get :unpublish
     post :unpublish
     post :unschedule
+    get :guidance
```

Comment:
Could this be included in the right place in this block to keep everything alphabetical?